cms/static/js/views/video_transcripts.js

```diff
@@ -23,6 +23,7 @@ define(
                 this.edxVideoID = options.edxVideoID;
                 this.clientVideoID = options.clientVideoID;
                 this.transcriptionStatus = options.transcriptionStatus;
+                this.errorDescription = options.errorDescription;
```

Comment:
Aren't we supposed to add errorDescription in videos_index.html too(as it will be available to JS only then)?

Reply:
we send complete information inside `previous_uploads` 
https://github.com/edx/edx-platform/blob/3b7918b07aabfc8a48677ac41a868af7fb0ea90b/cms/templates/videos_index.html#L74

Reply:
right